Source/Core/Core/IOS/IOS.cpp

```diff
@@ -137,6 +138,7 @@ static bool SetupMemory(u64 ios_title_id, MemorySetupType setup_type)
     return true;
   }
 
+  Memory::Memset(0, 0, 0x3fff);
```

Comment:
Should we introduce a few named constants for those (or at least leave a comment)?

Reply:
Sure, I've added named constants for the region address and size + a comment.